test/docker-compose.upgrade.yml

```diff
@@ -20,7 +20,7 @@ services:
       - MW_ADMIN_NAME=${MW_ADMIN_NAME}
       - MW_ADMIN_PASS=${MW_ADMIN_PASS}
       - MW_ADMIN_EMAIL=${MW_ADMIN_EMAIL}
-      - MW_WG_SECRET_KEY=${MW_SECRET_KEY}
+      - MW_WG_SECRET_KEY=${MW_WG_SECRET_KEY}
```

Comment:
Should this change be reverted too, so the "external variable name" remains `MW_SECRET_KEY` in call cases?